src/SearchResults.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import moment from "moment";
+
+const headings = [
+  "Id",
+  "Title",
+  "First Name",
+  "Surname",
+  "Email",
+  "room Id",
+  "Check-In",
+  "Check-Out",
+  "Night"
+];
+const SearchResults = props => {
+  return (
+    <table className="table">
+      <thead>
```

Comment:
Good table HTML 👏 
Later, we can look at how to add a <caption /> and hide it except for screen reader users, if required.